gfx/UltimateCataclysm/pngs_tall_32x64/layers/desk_laptop.json

```diff
@@ -0,0 +1,5 @@
+{
+  "id": "desk_laptop",
+  "fg": "desk_laptop",
```

Comment:
You don't need this json if the id is the name of the sprite, or am I missing something?

Reply:
No you don't thanks I'll remove it